i18n.js

```diff
@@ -4,6 +4,21 @@ const chalk = require('chalk');
 const get = require('lodash.get');
 const templite = require('templite');
 
+// should match "/en/", "/en-US/", "/en_US/"
+const contextLocaleRegex = /^\/([a-z]{2,}(?:[_-][a-z]{2,})?)\//i;
```

Comment:
Maybe maximum length for a segment should be limited to 2 exactly but there's a chance that somebody uses "eng" or "english" as their locale name so I left it as `[...]{2,}`